rest/model/data/meta.go

```diff
@@ -275,6 +285,12 @@ func MetaFromMap(m map[string]interface{}) *Meta {
 				} else {
 					fv.Set(reflect.ValueOf(ParseType(v.(string))))
 				}
+			case "Pulsar":
+				var pulsars []map[string]interface{}
+				err := json.Unmarshal([]byte(v.(string)), &pulsars)
```

Comment:
fwiw this could be `if err := .... ; err == nil`

Reply:
✔️